### DISCUSSION TITLE
Get Value of HTML Element

### DISCUSSION THREAD

**Discussion by itebco** [cefsharp/CefSharp#3662] [General]
HI everyone!
How can I get value of HTML Element
I try getElementsByClassName and getElementsById  but seem not working.

**Comment by amaitland** [comment]
Please read https://github.com/cefsharp/CefSharp/wiki/General-Usage#2-how-do-you-call-a-javascript-method-that-returns-a-result

You cannot directly return a htmlelement, there is a basic example in the wiki.

**Reply by itebco** [reply]
Many thanks for your comment!
